zgen.zsh

```diff
@@ -260,7 +260,7 @@ zgen-save() {
 
         local ages="$(stat -Lc "%Y" 2>/dev/null $ZGEN_RESET_ON_CHANGE || \
                       stat -Lf "%m" 2>/dev/null $ZGEN_RESET_ON_CHANGE)"
-        local shas="$(shasum -a 256 ${ZGEN_RESET_ON_CHANGE})"
+        local shas="$(sha256sum ${ZGEN_RESET_ON_CHANGE})"
```

Comment:
OS X doesn't ship with a `sha256sum` command.

Maybe check for `shasum` and `sha256sum` during start and put the shawhatever command into a variable and use that?